packages/embed/src/factories/create-popover/popover-options.ts

```diff
@@ -0,0 +1,54 @@
+import { ActionableOptions, UrlOptions, BehavioralOptions } from '../../base'
+
+export type PopoverOptions = UrlOptions &
+  ActionableOptions &
+  BehavioralOptions & {
+    /**
+     * Time until the embedded typeform will automatically close after a respondent clicks the Submit button. The default time is 5 seconds. PRO+ users can change the autoClose time.
+     *
+     * @type {number}
+     */
+    autoClose?: number
+    /**
+     * Hide fixed scrollbars.
+     *
+     * @type {boolean}
+     */
+    hideScrollbars?: boolean
+    /**
+     * Specify the width of the drawer or popover (only applies if using mode "drawer_left" or "drawer_right" or "popover" or "side_panel").
+     *
+     * @type {number}
+     */
+    width?: number
+    /**
+     * Specify the height of the popover (only applies if using mode "popover" or "side_panel").
+     *
+     * @type {number}
+     */
+    height?: number
+    /**
+     * Specify the size of the popover (only applies if using mode "popover").
+     *
+     * @type {number}
+     */
+    size?: number
+    /**
+     * Specify the color of the popover button
+     *
+     * @type {string}
+     */
+    buttonColor?: string
+    /**
+     * Specify the image url for the popover trigger button icon
+     *
+     * @type {string}
+     */
+    customIcon?: string
```

Comment:
We should also load those options from attributes in [loadOptionsFromAttributes](https://github.com/Typeform/embed/blob/6d7c673b0694318525c5b9774f885c319f29eb10/packages/embed/src/utils/load-options-from-attributes.ts#L58).

Reply:
Should they be included if they are popover specific?

Reply:
nvm now I understood how those options are loaded